REQUIRE

```diff
@@ -1,8 +1,9 @@
-julia 0.7-beta
+julia 0.7
 DynamicalSystemsBase 0.8.2
 StaticArrays 0.5.0
 StatsBase 0.8.2
 NearestNeighbors 0.1.0
 Reexport
 IterTools
 MultivariateStats
+PrincipalComponentAnalysis
```

Comment:
What is the status of the REQUIRE?

Reply:
Just saw your other comment, then its okay let's move on with MultivariateStats for now. Since everything is open we can still make our own in the future if need be.

Reply:
I updated the REQUIRE. I hope it's correct